templates/tidy_cron.epp

```diff
@@ -12,7 +12,7 @@ find "$DIR" -type f -ctime +$RETENTION_DAYS -delete
 
 #compress files
 cd "$DIR"
-find . -type f ! -name "*.bz2" > "$DIR.tmp";
-xargs -a "$DIR.tmp" tar -jcf "$DIR/$METRICS_TYPE-$(date +%Y.%m.%d.%H.%M.%S).tar.bz2" 2>/dev/null;
-xargs -a "$DIR.tmp" rm ;
-rm "$DIR.tmp";
+find manifests -type f -not -name "*.gz" -a -not -name "*.bz2" > "$DIR.tmp"
```

Comment:
how did this find get changed from `.` to `manifests`?  @danielparks  

In my testing it still needs to be `.`

Reply:
…

That's bizarre. There's nothing called manifests. Must have changed it accidentally somehow? I did test this…